components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImpl.java

```diff
@@ -2279,6 +2298,19 @@ private void addTransports(Connection connection, String apiID, Set<String> tran
         }
     }
 
+    private void addThreatProtectionPolicies(Connection connection, String apiId, Set<String> policies)
```

Comment:
Shall we add a method comment here?

Reply:
My mistake, I thought comments for private methods were not necessary. I'll add them. Thanks.